app/react/package.json

```diff
@@ -74,7 +74,8 @@
   },
   "devDependencies": {
     "@storybook/client-api": "6.5.0-alpha.21",
-    "@types/prompts": "^2.0.9"
+    "@types/prompts": "^2.0.9",
+    "webpack": "4"
```

Comment:
So we recently added webpack 4 - 5 as a dep here, and so yarn is eager to install webpack 5 here, we can force it to install webpack 4 (and then de-dup it to the root, by adding a devDependency that equal to the rest of the repo.